independent-projects/arc/tests/src/test/java/io/quarkus/arc/test/buildextension/observers/SyntheticObserverTest.java

```diff
@@ -52,13 +52,13 @@ public void register(RegistrationContext context) {
     @Test
     public void testSyntheticObserver() {
         MyObserver.EVENTS.clear();
-        Arc.container().beanManager().fireEvent("foo");
+        Arc.container().beanManager().getEvent().select(String.class).fire("foo");
```

Comment:
Hm, I think that the `.select(String.class)` part is unnecessary in these tests as `Arc.container().beanManager().getEvent().fire("foo")` is effectively the same as `Arc.container().beanManager().fireEvent("foo")`...

Reply:
True, I was just being explicit. Both approaches are equal and there is IMO no harm in either of them.

Reply:
Same here. Is calling `select` ever necessary, if you only pass it the event type? That type is discovered from the payload instance, right? (OK, maybe it's useful in case of parameterized types, I guess?)

Reply:
It is useful for type literal events as seen in `SimpleObserverInjectionTest` in this PR.
But since you both want it simplified, I will update the string-based events :)

Reply:
@mkouba @Ladicek removed the `select()` invocation where it was superfluous.

Reply:
OK, that's what I thought. Thanks!

Reply:
> It is useful for type literal events as seen in SimpleObserverInjectionTest in this PR.

It's not because the test observes the raw type (at least in `SimpleObserverInjectionTest.StringObserver#observeString()`).